core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -334,6 +335,9 @@ private TableProperties() {}
   public static final String MAX_REF_AGE_MS = "history.expire.max-ref-age-ms";
   public static final long MAX_REF_AGE_MS_DEFAULT = Long.MAX_VALUE;
 
+  public static final String DELETE_GRANULARITY = "write.delete.granularity";
+  public static final String DELETE_GRANULARITY_DEFAULT = DeleteGranularity.PARTITION.toString();
```

Comment:
Nit: I'd use a string so that we are forced to continue supporting it, like the other defaults. This would technically allow someone to change `PARTITION` in the code without breaking although it would change the property value.

Reply:
Ah, I see you override `toString` so it's probably fine.